### PR TITLE
bug fixes

### PR DESCRIPTION
**Pull request by lahiru823**

### CODE REVIEW

File: modules/jaggery-apps/app-publisher-web/controllers/logout.jag [hunk 1]
```diff
@@ -47,8 +47,8 @@
                     <%
         }
     }else {
-        session.invalidate(); //Clear the session
-        response.sendRedirect('/portal');
+        //this block executes when the session is expired so redirect user to login page
```
Comment (rushmin) [inline modules/jaggery-apps/app-publisher-web/controllers/logout.jag]: Comments should start with a capital letter.

File: modules/jaggery-apps/app-publisher-web/themes/appm/js/publisher.commons.js [hunk 2]
```diff
@@ -231,6 +231,11 @@ $(document).ready(function() {
 		var searchAssetString = $('#inp_searchAsset').val();
 		var searchPrefix = $('#search-prefix').val();
 
+		//set default when empty
```
Comment (rushmin) [inline modules/jaggery-apps/app-publisher-web/themes/appm/js/publisher.commons.js]: Comments should start with a capital letter.

File: modules/jaggery-apps/app-publisher-web/controllers/logout.jag (hunk 1 above)
Comment (lahiru823) [inline modules/jaggery-apps/app-publisher-web/controllers/logout.jag]: commit# https://github.com/wso2/carbon-appmgt/commit/508bfa570b41db3fdd27b95a7495c5a6c75f1929

File: modules/jaggery-apps/app-publisher-web/themes/appm/js/publisher.commons.js (hunk 2 above)
Comment (lahiru823) [inline modules/jaggery-apps/app-publisher-web/themes/appm/js/publisher.commons.js]: commit# https://github.com/wso2/carbon-appmgt/commit/508bfa570b41db3fdd27b95a7495c5a6c75f1929

**Pull request merged**